File: data/python/MINIMUM_ROTATIONS_REQUIRED_GET_STRING_TRANSLATED.java
====================
public static int f_gold ( String str ) {
  String tmp = str + str ;
  int n = str . length ( ) ;
  for ( int i = 1 ;
  i <= n ;
  i ++ ) {
    String substring = tmp . substring ( i , n ) ;
    if ( ( str . equals ( substring ) ) && ( str . equals ( substring ) ) ) {
      return i ;
    }
  }
  return n ;
}

File: data/java/MINIMUM_ROTATIONS_REQUIRED_GET_STRING_TRANSLATED.py
====================
def get_MINIMUM_ROTATIONS_REQUIRED_GET_STRING ( str ) :
    tmp = str + str
    n = len ( str )
    for i in range ( 1 , n + 1 ) :
        substring = tmp [ i : i + len ( str ) ]
        if str == substring :
            return i
    return n

